presets/disable-docker-datasource-for-postgres-and-redis.json

```
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "packageRules": [
+    {
+      "matchPackageNames": ["postgres", "redis"],
+      "matchDatasources": ["docker"],
+      "enabled": false
+    }
```

Au cas ou, cf le message de Scalingo qui dit que ce qui est sur le hub n'est pas forcement valide

```suggestion
    },
    {
      "matchDepTypes": ["postgres", "redis"],
      "matchDatasources": ["scalingo-postgres","scalingo-redis"],
      "automerge": false
    }
```

---

Done